README.md

```diff
@@ -23,7 +23,7 @@ Pyro is a flexible, scalable deep probabilistic programming library built on PyT
 Pyro is in an alpha release.  It is developed and used by [Uber AI Labs](http://uber.ai).
 For more information, check out our [blog post](http://eng.uber.com/pyro).
 
-## Installation
+## Installing Pyro stable release
```

Comment:
nit: i think it would be cleaner to have
```
## Installation
### Installing stable ...
### Installing from ...
``` 
so you can get the hyperlink to a specific section or the entire installation

Reply:
done